### DISCUSSION TITLE
Creating DDI of the drug

### DISCUSSION THREAD

**Discussion by Esther806** [Open-Systems-Pharmacology/Forum#843] [Questions & Problems] [answered]
![IMG_20201002_113745](https://user-images.githubusercontent.com/71115527/94885783-bbb03600-04a3-11eb-9890-877ece53b6bb.jpg)
This is my DDI model of theophylline and acyclovir. Am I did correctly?

**Comment by tobiasK2001** [comment]
Dear Esther,

from the screenshot it is hard to say. If the y-axis is linear (unfortunately it is cut of), it could be that Aciclovir is at zero. Did you check the administration event for aciclovir? 
Generally it would be easier to judge if you would provide you file. Can you attach it as zip file?

Best, Tobias

**Comment by Esther806** [comment]
![2020-10-02_14h39_32](https://user-images.githubusercontent.com/71115527/94895241-9e886100-04bd-11eb-8dca-aef80df653b8.png)
Acyclovir is given 50mg single dose and theophylline is given 250mg single dose. Both are given in oral route.

**Comment by tobiasK2001** [comment, marked as answer]
Dear Esther,

this it what it looks like for me, if I use your Info and just the default BB provided by PK-sim.
Aciclovir looks quite flat in the linear log scale view. 
![image](https://user-images.githubusercontent.com/25585248/94897998-12604480-0491-11eb-9e44-9d13a8f296ad.png)
Maybe it is better to log at log scale
![image](https://user-images.githubusercontent.com/25585248/94898193-6ec36400-0491-11eb-993f-fa95444a7b21.png)
B.t.w I prefer geomean and 5-95 % range for visualisation.
Of course your Simulation parameters will differ but at least your administration should look like this
![image](https://user-images.githubusercontent.com/25585248/94898320-a92d0100-0491-11eb-8feb-4e3e096ef4ed.png)
B.t.w. I am not sure if it makes sense to simulate a DDI with Acyclovir and Theophylline, as they do not seem to have a metabolism process in common, where an interaction could take place. However this note is just based on the inbuild templates. Maybe you have more information.
For more details please you can the attached file as example
[DDI-Acyclovir-Theophylline.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/5316519/DDI-Acyclovir-Theophylline.zip)
hope this might help.

Best, Tobias

**Comment by Esther806** [comment]
Thank you so much.
Let assume the model is correct.
According to the model, there is no interaction occur between these two medications. Can i say that?

Question 2:
![2020-10-02_15h43_34](https://user-images.githubusercontent.com/71115527/94899553-1f4b5b00-04c6-11eb-930e-335515f94e14.png)
acyclovir remain in 50mg single dose and digoxin is given 0.75mg single oral dose also.
i found few articles stated that no interaction occur between acyclovir and digoxin. But this is the model what i got. Am i did any mistake?

**Comment by tobiasK2001** [comment]
Dear Esther,

sorry for the late reply.

> According to the model, there is no interaction occur between these two medications. Can i say that?

Maybe it is a bit philosophic but I would rather phrase it like this: A model using building block templates provided by the software assumes no DDI between this two medications. However the model can tell us only what we put in and the information provided in the templates maybe out of date. A literature search for possible DDI mechanism between this to drugs maybe more conclusive to answer this question.

The same goes for you your second question (you might think about opening a new thread for this). Again hard to says from your picture, if there was made a mistake. I would generally to let relevant measured data judge if you model is OK. To check the technical setup of your model, again it is not possible without having the simulation file.

Best, Tobias
